app/src/main/java/com/github/se/icebreakrr/ui/sections/Settings.kt

```diff
@@ -101,8 +105,8 @@ fun SettingsScreen(
   val myProfile = profilesViewModel.selfProfile.collectAsState()
 
   Scaffold(
-      topBar = { TopBar("Settings") },
       modifier = Modifier.testTag("settingsScreen").fillMaxSize(),
+      topBar = { TopBar("Settings") },
```

Comment:
Nitpick: hardcoded string could be moved to strings.xml

Reply:
Yes I'll do it